app/models/my_module.rb

```diff
@@ -500,4 +500,12 @@ def uncomplete
   def create_blank_protocol
     protocols << Protocol.new_blank_for_module(self)
   end
+
+  def coordinates
```

Comment:
May be rename it to something like 'coordinates_validator' or 'coordinates_uniqueness_check' ?